### PR TITLE
Add CustomToggle component and its story

### PR DESCRIPTION
**Pull request by Dauntless-Jacobson**

### CODE REVIEW

**Comment by Dauntless-Jacobson** [comment]
toggle

**Comment by Dauntless-Jacobson** [comment]
togglewithcontrols

File: src/web-components/CustomToggle/index.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { LitElement, html, css } from "lit";
+import { customElement, property, state } from 'lit/decorators.js';
+
+/**
+ * `custom-select` Component Usage Guide
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:5]: `custom-toggle`

File: src/web-components/CustomToggle/index.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * 
+ * This component allows you to toggle the active state by passing the `isActive` property.
+ * 
+ * ### Properties:
+ * - **isActive**: (Boolean) A flag to toggle the active state of the component.
+ * 
+ * ### Example Usage:
+ * 
+ * ```html
+ * <custom-select isActive="${true}"></custom-select>
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:15]: `<custom-toggle .isActive="${true}"></custom-toggle>`

File: src/web-components/CustomToggle/index.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    render() {
+
+        const fillColor = this.isActive ? 'rgba(77, 166, 255, 0.15)' : 'rgba(255, 255, 255, 0.1)';
+        const pathData = this.isActive
+            ? 'M9 15h60a10 10 0 0 1 10 10v10a10 10 0 0 1-10 10H9a10 10 0 0 1-10-10V25a10 10 0 0 1 10-10z'
+            : 'M9 15h60a10 10 0 0 1 10 10v10a10 10 0 0 1-10 10H9a10 10 0 0 1-10-10V25a10 10 0 0 1 10-10z';
+        const circleCX = this.isActive ? '60' : '35';
+
+        return html`
+            <svg style="width:5.5rem; height: 2.5rem; display: flex; align-items: center;" viewBox="0 0 100 50" xmlns="http://www.w3.org/2000/svg">
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:32]: Let's add `cursor:pointer;` to the list of styles

File: src/web-components/CustomToggle/index.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+@customElement('custom-toggle')
+export class CustomToggle extends LitElement {
+    @property({ type: Boolean }) isActive = false;
+
+    render() {
+
+        const fillColor = this.isActive ? 'rgba(77, 166, 255, 0.15)' : 'rgba(255, 255, 255, 0.1)';
+        const pathData = this.isActive
+            ? 'M9 15h60a10 10 0 0 1 10 10v10a10 10 0 0 1-10 10H9a10 10 0 0 1-10-10V25a10 10 0 0 1 10-10z'
+            : 'M9 15h60a10 10 0 0 1 10 10v10a10 10 0 0 1-10 10H9a10 10 0 0 1-10-10V25a10 10 0 0 1 10-10z';
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:28]: This isn't being used.

File: src/web-components/CustomToggle/index.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * `custom-select` Component Usage Guide
+ * 
+ * This component allows you to toggle the active state by passing the `isActive` property.
+ * 
+ * ### Properties:
+ * - **isActive**: (Boolean) A flag to toggle the active state of the component.
+ * 
+ * ### Example Usage:
+ *
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:13]: Let's also add the following to the example usage:
```
const handleEvent = (event: Event) => {
        const toggleElement = event.currentTarget as CustomToggle;
        isActive = !isActive;
        toggleElement.isActive = isActive; // Set the reactive property on the custom element
    };
```

**Review by bigirishlion — CHANGES_REQUESTED** [review]
🏀

File: src/web-components/CustomToggle/index.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+ * 
+ * This component allows you to toggle the active state by passing the `isActive` property.
+ * 
+ * ### Properties:
+ * - **isActive**: (Boolean) A flag to toggle the active state of the component.
+ * 
+ * ### Example Usage:
+ * 
+ * ```html
+ * <custom-select isActive="${true}"></custom-select>
```
Comment (bigirishlion) [inline src/web-components/CustomToggle/index.ts:16]: It doesnt look like this was changed. It still says `<custom-select>`

**Review by bigirishlion — CHANGES_REQUESTED** [review]
Looks like just the comments `custom-select` needs to be changed still

**Review by bigirishlion — APPROVED** [review]
🔍

**Pull request merged**